.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -17,8 +17,8 @@ jobs:
 
   steps:
     - script: |
-        choco install cuda --version=10.2.89 --skip-powershell -fdv -y --debug
-      displayName: "Install Chocolatey Package: cuda --version=10.2.89 --skip-powershell"
+        choco install cuda --version=11.0.3 -fdv -y --debug
+      displayName: "Install Chocolatey Package: cuda --version=11.0.3"
```

Comment:
Noticed we are getting `11.0.3` instead of `10.2.89` for the `10.2` and `10.1` builds. Should these be updated?